### PR TITLE
Display a pod as terminating even if it isn't running

### LINKED ISSUE #1893: Pod status is wrong when pod is being terminated
**Describe the bug**

Pod shows "Container creating" status when pod is being terminated.

**To Reproduce**

Scale down deployment via terminal and watch pod list view/status column.

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

fixes #1893

### CODE REVIEW

File: src/renderer/api/endpoints/pods.api.ts [hunk 1]
```diff
@@ -342,7 +342,7 @@ export class Pod extends WorkloadKubeObject {
   // Returns pod phase or container error if occurred
   getStatusMessage() {
     if (this.getReason() === PodStatus.EVICTED) return "Evicted";
-    if (this.getStatus() === PodStatus.RUNNING && this.metadata.deletionTimestamp) return "Terminating";
+    if (this.metadata.deletionTimestamp) return "Terminating";
```
Comment (jim-docker) [inline src/renderer/api/endpoints/pods.api.ts:345]: with this it will probably never show "Terminated", which is checked for below.

Comment (Nokel81) [reply src/renderer/api/endpoints/pods.api.ts:345]: Even though we use the same root word ("terminate"), these two are actually different. This means that the pod has been chosen to be stopped by its parent deployment whereas the `state.terminated` means that the pod has run and then stopped. Which is why there is an `exitCode` field under `IPodContainerStatus.state.terminated`

Comment (jim-docker) [reply src/renderer/api/endpoints/pods.api.ts:345]: Ah, thanks. So when stopped by the parent deployment we just show "Terminating" until the pod disappears from the list?

**Review by jim-docker — APPROVED** [review]

File: src/renderer/api/endpoints/pods.api.ts (hunk 1 above)
Comment (Nokel81) [reply src/renderer/api/endpoints/pods.api.ts:345]: Yes, because that is what `kubectl` does.

**Review by jakolehm — APPROVED** [review]

**Pull request merged**